xonsh/platform.py

```diff
@@ -198,7 +192,7 @@ def win_ansi_support():
 
 @functools.lru_cache(1)
 def ptk_below_max_supported():
-    ptk_max_version_cutoff = (2, 0)
+    ptk_max_version_cutoff = (99999, 0)  # currently, no limit.
```

Comment:
Probably should be `(4, 0)`

Reply:
Hmm, whose ox should be gored?
if (4,0), then Xonsh team starts hearing from users that Xonsh is broken as soon as PTK team releases 4.0 and users accept the update.
if (bignum,0), then Xonsh team only hears about it if PTK team actually breaks V3 contract for the features Xonsh uses.
I don't know the history with PTK. They did seem to get it right this time, so I went for it.
But if you want to stick with (4,0), I'll set an upper bound in requirements.txt and setup.py to match.  You'll still start hearing from users when PTK 4 comes out, but only from the pioneers who want to use it and find it doesn't coexist with Xonsh.

Reply:
Yeah, I hear you. I guess in my experiance, the 1.0 & 2.0 transitions broke everything, so I was being a bit more conservative here